_slides/02-addyapproved.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-This is a new caption
+New caption
```

Comment:
Add more details to caption, it will help the other users

Reply:
Yes try it